src/authentication/service/token.service.ts

```diff
@@ -21,22 +21,20 @@ export class TokenService {
     }
     const token = this.authenticationHelper.generateTokenForUser(userRecord);
     token.refreshToken = refreshToken;
-    return token;
+    return { ...token, user: userRecord };
   }
 
   async resetToken(id: string): Promise<void> {
     await this.userService.updateField(id, 'refreshToken', '');
   }
 
   async getNewToken(userRecord: User): Promise<TokenResponse> {
-    const token: TokenResponse = this.authenticationHelper.generateTokenForUser(
-      userRecord,
-    );
+    const token = this.authenticationHelper.generateTokenForUser(userRecord);
     await this.userService.updateField(
       userRecord.id,
       'refreshToken',
       token.refreshToken,
     );
-    return token;
+    return { ...token, user: userRecord };
```

Comment:
Can we create a new variable of type TokenResponse and return it? Let's not skip 1 line of code compromising readability.